README.md

```diff
@@ -53,6 +53,7 @@ The configuration file uses the [TOML format](https://toml.io/). Below is an ove
 |                        | MOLLY_CONF                 | -c \*       | Path to the configuration file, optional          |                      | /etc/mollysocket.conf                                   |
 | host                   | MOLLY_HOST              \* |             | Listening address of the web server               | 127.0.0.1            | 0.0.0.0                                                 |
 | port                   | MOLLY_PORT              \* |             | Listening port of the web server                  | 8020                 | 8080                                                    |
+| webserver              | MOLLY_WEBSERVER         \* |             | Wether to start the web server (disable for air gapped mode) | true      | false                                                   |
```

Comment:
```suggestion
| webserver              | MOLLY_WEBSERVER         \* |             | Wether to start the web server | true      | false                                                   |
```

Web server doesn't have to be disable for air gapped mode, so this is misleading :)

Reply:
makes sense 👍